e2e/tests/002.sh

```diff
@@ -37,3 +37,14 @@ k8s_wait_exists "$kubeconfig" 600 "default" "cluster" "edge02"
 
 k8s_wait_ready "$kubeconfig" 600 "default" "cluster" "edge01"
 k8s_wait_ready "$kubeconfig" 600 "default" "cluster" "edge02"
+
+# Inter-connect worker nodes
+kubectl get secret regional-kubeconfig -o jsonpath='{.data.value}' | base64 -d > $HOME/.kube/regional-kubeconfig
```

Comment:
I have a [k8s_get_capi_kubeconfig](https://github.com/nephio-project/test-infra/blob/main/e2e/lib/k8s.sh#L108) function that will do this. It uses a tempfile right now but we could instead just put it in .kube.

Non-blocking, this is fine too.